ui/src/app/utils/leo-runtime-initializer.tsx

```diff
@@ -80,6 +82,7 @@ export interface LeoRuntimeInitializerOptions {
   overallTimeout?: number;
   maxCreateCount?: number;
   maxDeleteCount?: number;
+  maxPauseCount?: number;
```

Comment:
On further review, I don't think we need this at all. Just issue the Stop manually (you're already doing this, in runtime-utils), then use the initializer to wait for it to reach the Stopped status via the custom resolutionCondition. You'll also need to move the resolutionCond check higher in the list of checks, so we don't try to for example, `startRuntime` when we're actually done. The initializer itself has no need that I can see to invoke Stop.

Reply:
To expand on this slightly, the reason this is incongruent to the other options is that the others are all in service of moving the state machine towards a `Running` state - even though in several places this is more or less abused to exit early in another state, the state machine always moves towards Running. Adding pause as a knob here introduces ambiguity into the utility, e.g. maxPauseCount: 1, maxResumeCount: 1 is nonsensical. I think there's probably some more general refactoring to do here, but I think it shouldn't be necessary as part of this PR.